.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - checkout
       - run: git submodule sync
       - run: git submodule update --init
-      - run: echo 'docker run --pids-limit -1 --security-opt seccomp=unconfined --network host --user "$(id -u):$(id -g)" --rm -v $PWD:$PWD -w $PWD  docker.io/scylladb/seastar-toolchain:2022-06-14 "$@"' > run; chmod +x run
+      - run: echo 'docker run --pids-limit -1 --security-opt seccomp=unconfined --network host --user "$(id -u):$(id -g)" --rm -v $PWD:$PWD -w $PWD  docker.io/scylladb/seastar-toolchain:latest "$@"' > run; chmod +x run
```

Comment:
it seems `docker.io/scylladb/seastar-toolchain:latest` is not able to provide gcc-12 or clang-15.

Reply:
i see. probably we need to update the image on docker.

Reply:
Yes, I'll do that when committing.